src/test/regress/citus_tests/config.py

```diff
@@ -186,6 +186,7 @@ def __init__(self, arguments):
         self.new_settings = {
             "citus.use_citus_managed_tables": False,
         }
+        self.skip_tests = ["nested_execution"]
```

Comment:
We should maybe split the queries that returns correct results into a different file, say `nested_execution_supported` and only skip the ones that fail.

For the tests you added, I manually verified that PG gives the same results. But, in the future we can do that as well